rollup.config.js

```diff
@@ -2,10 +2,10 @@ import { terser } from 'rollup-plugin-terser'
 import babel from 'rollup-plugin-babel'
 
 export default {
-  input: './lib/script.js',
+  input: './src/script.js',
   output: [
     {
-      file: './lib/script.min.js',
+      file: './dist/script.min.js',
```

Comment:
Maybe we can support `min` flag for unbuild and unify tooling?

Reply:
Quick workaround: We can add `script` entry to `build` section in `package.json` and append `esbuild --minify dist/script.mjs --outfile=dist/script.min.js`